Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +14,37 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+const passwords = ["7wnuT$bH", "!uTd4*M", "456Ww^8"];
+
+function checkPassword(pass) {
+    if (pass.length < 5) {
+        return false;
+    } else if (!/[0-9]/.test(pass)) {
+        return false;
+    } else if (!/[A-Z]/.test(pass)) {
+        return false;
+    } else if (!/[a-z]/.test(pass)) {
+        return false;
+    } else if (!/[!#$%.&*]/.test(pass)) {
+        return false;
+    } else if (passwords.includes(pass)) {
+        return false;
+    }
+    return true;
+}
+
+let userPassword = checkPassword("te^A");
+// console.log(userPassword);
+
+// jest tests 
+test("check password", () => {
```

Comment:
Again, really want some explanations in your tests. Imagine your colleague doesn't know the password rules, but they've come to this program to find out why a failing test is taking down your application. How many steps would it take them to find out that 

`expect(checkPassword("te^")).toBe(false);` is failing BECAUSE it's too short. 

They shouldn't have to read outside the test to figure this out.